integration/network/ipvlan/ipvlan_test.go

```diff
@@ -425,7 +428,21 @@ func testIpvlanAddressing(client dclient.APIClient) func(*testing.T) {
 	}
 }
 
-// ensure Kernel version is >= v4.2 for ipvlan support
+// ensure ipvlan is supported and enabled
```

Comment:
This comment is mildly misleading. The code (in its current form) does not ensures anything, it merely checks for a feature presence.

Reply:
Fair point.